lib/routes.php

```diff
@@ -15,9 +15,12 @@
 
 // /{pagename}.{representation}
 if ($representation != '') {
-  if (!$page_exists) {
+  if (!$page_exists) {	  
     header('HTTP/1.0 404 Not Found');
-    die("Page not found: <a href=".EDITH_URI."/".$page->name.">Create me!</a>");
+    
+    // generate 404 response by utilizing my 404 page (html)
+    include("404.php");
```

Comment:
Since `404.php` is a template like `html.php`, we should probably move the `404.php` inside the `templates` folder instead of inside `lib`.